backend/src/main/java/com/cruru/applicant/domain/EvaluationStatus.java

```diff
@@ -1,6 +1,28 @@
 package com.cruru.applicant.domain;
 
+import com.cruru.applicant.domain.dto.ApplicantCard;
+import com.cruru.applicant.exception.badrequest.EvaluationStatusException;
+import java.util.Arrays;
+import java.util.function.Predicate;
+
 public enum EvaluationStatus {
 
-    ALL, NO_EVALUATED, EVALUATED
+    ALL(card -> true),
+    NOT_EVALUATED(card -> card.evaluationCount() == 0),
+    EVALUATED(card -> card.evaluationCount() > 0);
```

Comment:
코드를 더 보니 ApplicantCard에도 책임을 더 부여할 수 있다는 생각이 드네요! 

```suggestion
    NOT_EVALUATED(card -> !card.hasEvaluation()),
    EVALUATED(ApplicantCard::hasEvaluation);
```

아니면 요런식으로?!

```suggestion
    NOT_EVALUATED(ApplicantCard::hasNoEvaluation),
    EVALUATED(ApplicantCard::hasEvaluation);
```

Reply:
정말 👍
반영했습니다.